app/assets/stylesheets/pageflow/print_view.css.scss

```diff
@@ -112,32 +109,31 @@ body {
     }
 
     h1, h2, h3, p {
-    max-width: initial;
+      max-width: initial;
     }
 
-
     .backgroundArea {
       width: initial;
       height: initial;
       position: absolute;
       background-color: white;
     }
-    .scroller, .content .scroller {
+    .scroller, .content .scroller  {
       opacity: 1 !important;
       visibility: visible !important;
       height: auto;
       position: relative;
     }
     .scroller > div {
-      transform: translate3d(0,0,0) !important;
-      -ms-transform: translate3d(0,0,0) !important; /* IE 9 */
-      -webkit-transform: translate3d(0,0,0) !important; /* Safari and Chrome */
-      transform: translate(0,0) !important;
-      -ms-transform: translate(0,0) !important; /* IE 9 */
-      -webkit-transform: translate(0,0) !important; /* Safari and Chrome */
+      transform: translate3d(0, 0, 0) !important;
+      -ms-transform: translate3d(0, 0, 0) !important; /* IE 9 */
+      -webkit-transform: translate3d(0, 0, 0) !important; /* Safari and Chrome */
+      transform: translate(0, 0) !important;
```

Comment:
Property `transform` already defined on line 128<br>!important should not be used